MediaBrowserTv/app/src/main/java/tv/emby/embyatv/TvApp.java

```diff
@@ -417,18 +417,23 @@ public void setLastUserInteraction(long lastUserInteraction) {
     }
 
     public boolean checkPaidCache() {
+        /*
```

Comment:
I think it's better to remove the old code (that is never going to be used) altogether, commented out code is hard to catch in diff.

Reply:
Resolved via c3c7b95